README.md

```diff
@@ -30,6 +30,11 @@ There are lots of ways to contribute, not just writing code!
 
 See our [Code of Conduct](CODE_OF_CONDUCT.md) and our [Contributor Guide](CONTRIBUTING.md) to learn more about how we work together as a community and how you can contribute.
 
+## :book: Documentation
+
+Full documentation is available at the [Data Safe Haven website](https://alan-turing-institute.github.io/data-safe-haven).
+If you think anything there is unclear, please consider reading how to contribute a documentation change in our [Contributor Guide](CONTRIBUTING.md#making-a-change-to-documentation).
```

Comment:
Do we actually need this section at all? The docs are already linked from the `Community & support` section above, while how to contribute is covered in the `Contributing` section.

Reply:
I'm happy to remove it